### PR TITLE
fix(demoapp): Replaced non-existent web images with locals

### PR DESCRIPTION
**Pull request by igor-sirotin**
The [original image]("https://pbs.twimg.com/profile_images/1369221718338895873/T_5fny6o_400x400.jpg") doesn't exist anymore. Replaced it with another local image.

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [ ] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

**Review by caybro — APPROVED** [review]

**Review by osmaczko — APPROVED** [review]

**Review by MishkaRogachev — CHANGES_REQUESTED** [review]

File: sandbox/controls/Controls.qml [hunk 1]
```diff
@@ -32,14 +32,14 @@ GridLayout {
     }
 
     StatusIconTabButton {
-        icon.source: "https://pbs.twimg.com/profile_images/1369221718338895873/T_5fny6o_400x400.jpg"
+        icon.source: "qrc:/demoapp/data/profile-image-1.jpeg"
```
[the remark this answers is not in the document]
Comment (caybro) [reply sandbox/controls/Controls.qml:35]: Not true, `ListElement` does support dynamic values too

[the remark this answers is not in the document]
Comment (caybro) [reply sandbox/controls/Controls.qml:35]: > Beginning with Qt 5.11 ListElement also allows assigning a function declaration to a role. This allows the definition of ListElements with callable actions.

See https://doc.qt.io/qt-5/qml-qtqml-models-listelement.html

**Review by PascalPrecht — CHANGES_REQUESTED** [review]
@igor-sirotin So to give some context here: There are some image sources in the model here that **meant to** return a 404, so that the components that make use of it, fallback to something else.

For example, in the case of list item profile pictures, if the picture can't be loaded for whatever reason, the letter identicon will be shown.

With that said, I'm not sure this PR can be merged because by maybe all of those image sources local, we no longer test the error case.

File: sandbox/controls/Controls.qml (hunk 1 above)
[the remark this answers is not in the document]
Comment (caybro) [reply sandbox/controls/Controls.qml:35]: Right, it's still an improvement, as now you can do things like:
```qml
ListElement { text: qsTr("Translate me!") }
```
which were not possible before :)

File: sandbox/controls/Controls.qml [hunk 2]
```diff
@@ -32,14 +32,14 @@ GridLayout {
...
     }
 
     StatusIconTabButton {
         icon.color: Theme.palette.miscColor9
         // This icon source is flawed and demonstrates the fallback case
         // when the image source can't be loaded
-        icon.source: "https://pbs.twimg.com/profile_images/1369221718338895873/T_5fny6o_400x400.jp"
+        icon.source: "qrc:/demoapp/data/profile-image-1.jpeg"
```
Comment (PascalPrecht) [inline sandbox/controls/Controls.qml:42]: @igor-sirotin This one has a comment that states the fallback

Comment (PascalPrecht) [reply sandbox/controls/Controls.qml:42]: Actually, seems like only this one has a broken URL on purpose

**Review by PascalPrecht — COMMENTED** [review]
@igor-sirotin Seems like there's only one spot where we do the fallback.
Feel free to change the endpoint to something else if you think it's necessary.
Otherwise, this PR can prolly be merged.

**Review by MishkaRogachev — APPROVED** [review]
LGTM!

**Review by PascalPrecht — APPROVED** [review]

**Pull request merged**